fiftyone/core/annotation/validate_label_schema.py

```
@@ -0,0 +1,529 @@
+                raise ValueError(
+                    "'precision' and 'values' are incompatible settings for "
+                    f"field '{field_name}'"
+                )
+            _validate_precision(field_name, value)
+        elif key == foac.READ_ONLY:
+            _validate_read_only(field_name, value)
+        elif key == foac.TYPE and value != _str_type:
+            _raise_type_error(field, field_name, value)
+
+
+def _validate_float_int_list_field_label_schema(
```

Pylint (4.0.3)

[convention] Function name "_validate_float_int_list_field_label_schema" doesn't conform to '[a-z_]([a-z0-9_]{0,30})$' pattern

(C0103)